internal/fs/wrappers/monitoring.go

```diff
@@ -327,9 +329,17 @@ func (fs *monitoring) Destroy() {
 type wrappedCall func() error
 
 func invokeWrapped(ctx context.Context, opName string, w wrappedCall) error {
+	// SpanKind is server since Fuse is a server for Kernel requests.
```

Comment:
I think we can have this flag protected. Considering this is going to get invoked for every fs call. 
Is the sampling set in the client code or in the exporter or in cloud tracer. 
Would the flag protection be added after config cli changes merged?

Reply:
Discussed offline. Without the TracerProvider its a no-op. 
Discussed to do execute-perf-test and share results.